src/charm.py

```diff
@@ -140,6 +146,27 @@ def _on_alert_rules_changed(self, _):
             prometheus_relation = self.model.get_relation("prometheus")
             prometheus_relation.data[self.app]["alert_rules"] = json.dumps(alert_rules_content)  # type: ignore[union-attr]  # noqa: E501
 
+    def _on_prometheus_configurer_relation_joined(self, event: RelationJoinedEvent) -> None:
+        """Handles actions taken when Prometheus Configurer relation joins.
+
+        Returns:
+            None
+        """
+        self._add_service_info_to_relation_data_bag(event)
+
+    def _add_service_info_to_relation_data_bag(self, event: RelationJoinedEvent) -> None:
+        """Adds information about Prometheus Configurer service name and port to relation data bag.
+
+        Returns:
+            None
+        """
+        prometheus_configurer_relation = event.relation
+        prometheus_configurer_relation.data[self.app][
+            "service_name"
+        ] = self.PROMETHEUS_CONFIGURER_SERVICE_NAME
+        prometheus_configurer_relation.data[self.app]["port"] = self.PROMETHEUS_CONFIGURER_PORT
+        print(prometheus_configurer_relation.data)
```

Comment:
If this is still needed, can you change it to a `logger` call? If not, please remove it.